eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -767,22 +792,29 @@ instance
   ( Era era,
     ToCBOR (Core.TxOut era),
     ToCBOR (Core.PParams era),
-    ToCBOR (State (Core.EraRule "PPUP" era))
+    ToCBOR (State (Core.EraRule "PPUP" era)),
+    ToCBOR (StashedAVVMAddresses era)
   ) =>
   ToCBOR (NewEpochState era)
   where
-  toCBOR (NewEpochState e bp bc es ru pd) =
-    encodeListLen 6 <> toCBOR e <> toCBOR bp <> toCBOR bc <> toCBOR es
+  toCBOR (NewEpochState e bp bc es ru pd av) =
```

Comment:
Alternatively, we could only change the serialization for eras for which this is not `()`. That way today's users would never have to reapply from the genesis block. (My goal is just to avoid an unnecessary long replay for users, but that's ultimately for a different decision maker to balance.)

Reply:
I considered that, but there are already serialisation changes in the ledger which will be visible in the next release, so there's not much point.